src/OSPSuite.Presentation/Presenters/SimulationPredictedVsObservedChartPresenter.cs

```diff
@@ -87,7 +87,8 @@ protected override void AddRunResultToChart()
          addPredictedVsObservedToChart(_simulation.ResultsDataRepository, (column, curve) =>
          {
             curve.Description = curve.Name;
-            curve.Name = column.PathAsString;
```

Comment:
The curve name shouldn't be overridden in this case because all the curves get the name of the calculation DataColumn